src/mono/netcore/System.Private.CoreLib/src/System/String.Mono.cs

```diff
@@ -9,7 +9,7 @@ namespace System
     public partial class String
     {
         [Intrinsic]
-        public static readonly string Empty;
+        public static readonly string Empty = null!;
```

Comment:
Doesn't even `=null` for a static field adds a static ctor?

Reply:
CoreCLR's implementation for this field is wrapped with `#pragma warning disable CS8618 // compiler sees this non-nullable static string as uninitialized`

Reply:
> Doesn't it add a static ctor?

Yes (https://github.com/dotnet/roslyn/issues/42985).

> #pragma warning disable CS8618

I can do the same thing here.

Reply:
That's not great, perhaps we can pragma the nullable warning untill it's fixed

Reply:
What's not great, the static cctor?  If so, I already changed it to use a pragma ;)

Reply:
Yeah, the empty static ctor.

>If so, I already changed it to use a pragma 

I missed that

Reply:
:smile: